stix_shifter_modules/arcsight/stix_translation/json/stix_2_1/from_stix_map.json

```diff
@@ -0,0 +1,158 @@
+{
+  "ipv4-addr": {
+    "fields": {
+      "value": ["sourceAddress", "destinationAddress"]
+    }
+  },
+  "ipv6-addr": {
+    "fields": {
+      "value": ["fulltextSearch"]
+    }
+  },
+  "mac-addr": {
+    "fields": {
+      "value": ["sourceMacAddress", "destinationMacAddress"]
+    }
+  },
+  "network-traffic": {
+    "fields": {
+      "src_port": ["sourcePort"],
+      "dst_port": ["destinationPort"],
+      "protocols[*]": ["transportProtocol", "applicationProtocol"],
+      "src_ref.value": ["sourceAddress", "sourceMacAddress"],
+      "dst_ref.value": ["destinationAddress", "destinationMacAddress"]
+    }
+  },
+  "directory": {
+    "fields": {
+      "path": ["filePath"]
+    }
+  },
+  "file": {
+    "fields": {
+      "parent_directory_ref.path": ["filePath"],
+      "name": ["fileName"],
+      "hashes.'SHA-256'": ["fulltextSearch"],
+      "hashes.'SHA-1'": ["fulltextSearch"],
+      "hashes.MD5": ["fulltextSearch"]
+    }
+  },
+   "process" : {
+    "fields": {
+      "name": ["destinationProcessName", "sourceProcessName"],
```

Comment:
Process name was deprecated for STIX 2.1

Reply:
removed